### ISSUE TITLE
Source mCAD files

### ISSUE THREAD

**Issue by ifohancroft** [ploopyco/thumb-trackball#5]
Would you please upload the source mechanical CAD files to the repo as well?

**Comment by ifohancroft** [comment]
This is not completed. The source files for the mechanical parts are still missing. There are still just STL and STEP files in the mechanical part of the repo.

Was this meant to be a "WON'T FIX"?

**Comment by keyboard-magpie** [comment]
A STEP file is a valid CAD source file. There should be no onus on ploopy to release deeper CAD files.

**Comment by ifohancroft** [comment]
Ok, let me rephrase then - the parametric CAD files.
Also, I know they are not required to release the mechanical parts as open source. I am just making a request for them to do so.
